_gettingstarted/4-python-highcharts-core.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: article
+title: Highcharts-core
+description: A comprehensive guide on how to work with Highchats-core, a Python library for creating dynamic charts.
+tags: [highcharts-core, charts, python, pip]
+image: https://learn.piptrends.com/assets/images/getting-started/getting-started-python-docs-cover.png
+cover: /assets/images/getting-started/Python-docx-Icon.png
+article_header:
+  type: overlay
+  theme: dark
+  background_image:
+  gradient: 'linear-gradient(135deg, #82d2a6, #8087e8)'
+date: 2023-12-01
+author: Lavanya
+---
+
+## Introduction
+
+In the world of data analysis and visualization, it is critical to convey information in a compelling and insightful manner. Highcharts, a popular JavaScript library, is a powerful tool for creating dynamic and interactive charts. What if you could use Highcharts' power directly in your Python projects with a simple 'pip install' command? 'highcharts-core' is a Python package that seamlessly integrates Highcharts functionality into your Python environment.
+
+## What is `highcharts-core`?
```

Comment:
no need to add `
it'll look like this 
![image](https://github.com/tankala/learn-pip-trends/assets/67141217/9de8f25d-a48f-47cf-972c-ad0fb3661868)